test/maybe.test.js

```diff
@@ -0,0 +1,180 @@
+const maybe = require("monio/maybe");
+const just = require("monio/just");
+const qunit = require("qunit");
+const { identity, inc, twice, justProp, maybeProp } = require("./utils");
+
+qunit.module("maybe");
+
+qunit.test("#unit", (assert) => {
+	assert.equal(
+		maybe(1)._inspect(),
+		"Maybe:Just(1)",
+		"should create a Maybe monad via monad"
```

Comment:
tiny nitpick, but this description (as well as others like it in the other tests) could possibly be clearer with "...via constructor" (instead of "...via monad"), to describe the creation of the monad via the "type constructor function" (even though `new` isn't used).